accel_code/utils.py

```diff
@@ -21,8 +20,9 @@ def get_lab_id(old_file_name):
 
 # This function returns the root data path for storing project data
 def get_test_data_path(project):
-    project_dir = os.path.join('\\vosslabhpc\Projects', project)
+
+    project_dir = os.path.join('vosslabhpc', 'Projects', project)
```

Comment:
Depending on the project, we will need logic to traverse to the correct directory.

For example:
```
if project == "BETTER":
    bids_dir = os.path.join(....)
elif project == "BikeExtend":
    bids_dir = os.path.join(...)

etc...
```